sql/src/main/java/io/crate/operation/collect/LuceneDocCollector.java

```diff
@@ -60,6 +49,8 @@
  */
 public class LuceneDocCollector extends Collector implements CrateCollector, RowUpstream {
 
+    public final static int PAGE_SIZE = 10000;
```

Comment:
isn't there a better place for the PAGE_SIZE? It shouldn't be Lucene specific, right?